testing/MLDB-1003-s3-load-dataset.js

```diff
@@ -17,7 +17,7 @@ function assertEqual(expr, val, msg)
 csv_conf = {
     type: "import.text",
     params: {
-        dataFileUrl : "s3://benchm-ml--main/test.csv",
+        dataFileUrl : "s3://dev.mldb.datacratic.com/testing/MLDB-1003/test.csv",
```

Comment:
Actually, there is something strange here.  I cannot access that public bucket s3://benchm-ml--main/test.csv (Nicolas neither) with my credentials but jenkins can.  My solution was to move the file to the dev.mldb.datacratic.com bucket but if we can fix it differently that would be great.  Any ideas?

Regarding accessing private resources in an open-source project, I've created MLDBFB-484 earlier this week because this concerns me as well.